src/main/java/pages/HomePage.java

```diff
@@ -9,5 +10,38 @@ public HomePage(WebDriver driver) {
         super(driver);
     }
 
+    private final By HomePageDisplay = By.xpath("//a[contains(text(),'Home')]");
+    private final By Signuplink = By.xpath("//a[contains(text(),'Signup / Login')]");
 
+    public String VerifyHomePageDisplay() {
+        return driver.findElement(HomePageDisplay).getText();
+    }
+
+    public SignUpPage clickSignuplink() {
+        driver.findElement(Signuplink).click();
+        return new SignUpPage(driver);
+    }
+
+    final static By loginLink = By.partialLinkText("Signup / Log");
```

Comment:
remove static and add private in all By object locator

Reply:
please check the comment @EmanMonem